.circleci/config.yml

```
@@ -76,6 +76,12 @@ jobs:
       - run:
           name: Confirm Version Updated
           command: node -e "if (require('./package.json').version === '0.0.0-development') process.exit(1)"
+      - run:
+          name: Set NPM Credentials
+          command: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc
```

```suggestion
          command: echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > ~/.npmrc
```

It's good practice to not put authenticated `npmrc` files in the same folder that is being published